src/System.Runtime.Extensions/src/System/Environment.Win32.cs

```diff
@@ -341,13 +342,17 @@ public static string UserName
                 }
 
                 return string.Empty;
+#else
+                return "Windows User";
+#endif
```

Comment:
@stephentoub didn't know about partial methods 😉 will merge files and do something like that